boxwise_flask/models/language.py

```diff
@@ -0,0 +1,17 @@
+from peewee import SQL, CharField, IntegerField
```

Comment:
I do not think we need this whole file

Reply:
Can I remove this from the user table also?